pkg/types/baremetal/validation/platform.go

```diff
@@ -115,6 +116,43 @@ func validateHosts(hosts []*baremetal.Host, fldPath *field.Path) field.ErrorList
 	return hostErrs
 }
 
+func validatePlatformExt(p *baremetal.Platform, fldPath *field.Path) field.ErrorList {
```

Comment:
please rename this function to be more readable.

Reply:
Done. In future I think it would be better to have just one generic method for all the validations performed via annotatons (potentially, `ValidatePlatform` itself)